inference-engine/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp

```diff
@@ -0,0 +1,222 @@
+        } else {
+            isaType = "sse42";
+        }
+        selectedType = "jit_" + isaType + "_" + "FP32";
+
+        std::vector<size_t> inputShape1, inputShape2;
+        if (inputShapes.size() == 1) {
+            inputShape1 = inputShape2 = inputShapes.front();
+        } else if (inputShapes.size() == 2) {
+            inputShape1 = inputShapes.front();
+            inputShape2 = inputShapes.back();
+        } else {
+            THROW_IE_EXCEPTION << "Incorrect number of input shapes";
+        }
+
+        configuration.insert(additional_config.begin(), additional_config.end());
+        auto input = ngraph::builder::makeParams(ngPrc, {inputShape1});
+
+        std::vector<size_t> shape_input_secondary;
+        switch (opType) {
+            case CommonTestUtils::OpType::SCALAR: {
+                shape_input_secondary = std::vector<size_t>({1});
+                break;
+            }
+            case CommonTestUtils::OpType::VECTOR:
+                shape_input_secondary = inputShape2;
+                break;
+            default:
+                FAIL() << "Unsupported Secondary operation type";
+        }
+
+        std::shared_ptr<ngraph::Node> secondaryInput;
+        if (eltwiseType == ngraph::helpers::EltwiseTypes::DIVIDE ||
+            eltwiseType == ngraph::helpers::EltwiseTypes::FLOOR_MOD ||
+            eltwiseType == ngraph::helpers::EltwiseTypes::MOD) {
+            std::vector<float> data(ngraph::shape_size(shape_input_secondary));
+            data = NGraphFunctions::Utils::generateVector<ngraph::element::Type_t::f32>(ngraph::shape_size(shape_input_secondary));
+            for (float &i : data) {
+                if (i == 0) {
+                    i = 1;
+                }
+            }
+            secondaryInput = ngraph::builder::makeConstant(ngPrc, shape_input_secondary, data);
+        } else {
+            secondaryInput = ngraph::builder::makeInputLayer(ngPrc, secondaryInputType, shape_input_secondary);
+            if (secondaryInputType == ngraph::helpers::InputLayerType::PARAMETER) {
+                input.push_back(std::dynamic_pointer_cast<ngraph::opset3::Parameter>(secondaryInput));
+            }
+        }
+
+        auto eltwise = ngraph::builder::makeEltwise(input[0], secondaryInput, eltwiseType);
+        eltwise->get_rt_info() = CPUTestsBase::setCPUInfo(inFmts, outFmts, priority);
+        function = std::make_shared<ngraph::Function>(eltwise, input, "Eltwise");
+    }
+
+    std::vector<cpu_memory_format_t> inFmts, outFmts;
+    std::vector<std::string> priority;
+    std::string selectedType;
+};
+
+TEST_P(EltwiseLayerCPUTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    Run();
+    CheckCPUImpl(executableNetwork, "Eltwise", inFmts, outFmts, selectedType);
+}
+
+namespace {
+
+std::vector<std::vector<cpu_memory_format_t>> filterCPUMemoryFormat(int count) {
+    std::vector<std::vector<cpu_memory_format_t>> resCPUMemoryFormats;
+
+
+
+    return resCPUMemoryFormats;
```

Comment:
Is it normal that `filterCPUMemoryFormat()` return empty `resCPUMemoryFormats`?

Reply:
Fixed